File: MangoAPI.Client/src/app/types/notifications/PrivateChatCreatedNotification.ts
import { CommunityType } from '../enums/CommunityType';

export interface PrivateChatCreatedNotification {
  chatId: string;
  title: string;
  communityType: CommunityType;
  description: string;
  membersCount: number;
  isArchived: boolean;
  isMember: boolean;
  chatLogoImageUrl: string;
}

File: MangoAPI.BusinessLogic/Notifications/PrivateChatCreatedNotification.cs
using MangoAPI.Domain.Enums;
using System;

namespace MangoAPI.BusinessLogic.Notifications;

public record PrivateChatCreatedNotification(
    Guid ChatId,
    string Title,
    CommunityType CommunityType,
    string Description,
    int MembersCount,
    bool IsArchived,
    bool IsMember,
    string ChatLogoImageUrl);